client/db_test.go

```diff
@@ -266,7 +266,7 @@ func ExampleTx_Commit() {
 	// 1/0: ab=2
 }
 
-func ExampleDB_Insecure() {
+func ExampleDB_insecure() {
```

Comment:
missed this earlier. Prepared squashed new PR, will close this PR and open a new one.